ask-sdk-dynamodb-persistence-adapter/lib/attributes/persistence/PartitionKeyGenerators.ts

```diff
@@ -62,4 +62,21 @@ export const PartitionKeyGenerators = {
 
         return requestEnvelope.context.System.device.deviceId;
     },
+
+    /**
+     * Gets attributes id using person id.
+     * @param {RequestEnvelope} requestEnvelope
+     * @returns {string}
+     */
+    personId(requestEnvelope : RequestEnvelope) : string {
+        if (requestEnvelope
+              && requestEnvelope.context
+              && requestEnvelope.context.System
+              && requestEnvelope.context.System.person
+              && requestEnvelope.context.System.person.personId) { 
+            return requestEnvelope.context.System.person.personId;
+        }
+
+        return this.userId(requestEnvelope);
```

Comment:
According to our tslint rules, the "this" keyword is disallowed outside of a class body

Reply:
A little concerned about backward compatibility if we were to change this file to a class format. Is there any other way of achieving this?